bin/redshift/setup_aws_redshift.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
```

Comment:
- no need for `pwd`, use parameter replacements like `cd "${BASH_SOURCE%/*}"`
- why `[0]`? is BASH_SOURCE an array?

Reply:
> is `BASH_SOURCE` an array?

Yes

https://www.gnu.org/software/bash/manual/html_node/Bash-Variables.html#index-BASH_005fSOURCE